mappings/net/minecraft/world/gen/WorldProvider.mapping

```diff
@@ -1,6 +1,8 @@
 CLASS none/auf net/minecraft/world/gen/WorldProvider
 	FIELD b world Lnone/aiw;
 	FIELD c biomeProvider Lnone/ajn;
+	FIELD d waterEvaporates Z
+	FIELD e noSky Z
```

Comment:
I think inverted booleans are more a preference than a rule, I use them when it makes sense to. `underworld` doesn't describe the End or imply that this has to do with calculations involving the sky.